spec/intl.html

```diff
@@ -855,11 +855,14 @@ <h1>
             1. If _required_ is ~time~ or ~any~, then
               1. Set _formatOptions_.[[hourCycle]] to _options_.[[hourCycle]].
           1. Let _needDefaults_ be *true*.
+          1. Let _anyPresent_ be *false*.
           1. For each property name _prop_ of _requiredOptions_, do
             1. Let _value_ be _options_.[[&lt;_prop_>]].
             1. If _value_ is not *undefined*, then
               1. Set _formatOptions_.[[&lt;_prop_>]] to _value_.
               1. Set _needDefaults_ to *false*.
+              1. Set _anyPresent_ to *true*.
+          1. If _anyPresent_ is *false*, return *null*.
```

Comment:
Just doing this will break `new Intl.DateTimeFormat().format()`, won't it?

Reply:
Quite right. The logic I intended was to return null if you specify any of the date/time element properties but none of the relevant ones. Fixed.